lm75bd/lm75bd.c

```diff
@@ -27,6 +27,29 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
+  // Internal temperature register: 00000000 or 0x00U
+  if (temp == NULL) return ERR_CODE_INVALID_ARG;
+  
+  error_code_t errCode;
+
+  uint8_t tempRegister[1U] = {0}; // makes all elements of the array 0
+  tempRegister[0] = 0x00U; // sets the internal temperature register
+  errCode = i2cSendTo(LM75BD_OBC_I2C_ADDR, tempRegister, 1U);
+
+  LOG_ERROR_CODE(errCode);
```

Comment:
if the errCode was `ERR_CODE_SUCCESS` it doesn't make sense to log that since it'll be harder to detect the actual errors if the log is full of success codes

Reply:
Also we shouldn't be continuing with the function and returning `ERR_CODE_SUCCESS` eventually if there was an error here, so just use `RETURN_IF_ERROR_CODE` when calling the i2c functions instead